src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/AttendanceService.kt

```diff
@@ -215,6 +215,16 @@ open class AttendanceService(
         .collect(Collectors.toSet())
   }
 
+  @Transactional
+  open fun deleteAttendances(offenderNo: String) {
+    val bookingId = elite2ApiService.getOffenderBookingId(offenderNo)
+    val attendances = attendanceRepository.findByBookingId(bookingId)
+
+    log.info("Deleting the following attendance records ${attendances.map { it.id }.joinToString(",")}")
```

Comment:
One for the next PR - I think we agreed that we should raise a telemetry event for the deletes for an offender too, but can't remember the exact format off the top of my head.  The idea is then that we can query app insights to see that the delete has filtered down to all systems